Atlas/android/app/src/main/java/com/bounswe2017/group10/atlas/response/OnCreateItemResponse.java

```diff
@@ -48,41 +47,27 @@ public void onResponse(Call<CreateItemResponse> call, Response<CreateItemRespons
             } else {
                 int id = response.body().getId();
 
-                RandomStringGenerator generator = new RandomStringGenerator.Builder()
-                        .withinRange('a', 'z')
-                        .build();
-                List<Image> localImages = new ArrayList<>();
-                List<String> filenames = new ArrayList<>();
-                for (Image img : this.mImageList) {
-                    if (Utils.isLocalUrl(img.getUrl())) {
-                        filenames.add(img.getUrl());
-
-                        String filename = generator.generate(Constants.CLOUDINARY_IMG_NAME_LENGTH);
-                        img.setUrl(Utils.filenameToCloudinaryUrl(filename));
-                        localImages.add(img);
+                int lastLocalUrl = 0;
```

Comment:
Wouldn't it better to name this as lastLocalUrlIndex, it certainly is an integer so the name is confusing.

Reply:
fixed